.pre-commit-config.yaml

```diff
@@ -1,5 +1,6 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 19.3b0
+    rev: 19.10b0
```

Comment:
👍 even if I'm not really a fan of freezing things but I welcome this change if it means we'll have less problems with formatting in our PRs

Reply:
Or we keep it for now and then we'll see?